nativeApp/src/lib/ui/forms/DateTimePicker/index.tsx

```diff
@@ -69,27 +69,27 @@ const DateTimePicker: FC<Props> = (props) => {
   const {
     onConfirm, value, mode, label, minDate, maxDate,
   } = props;
-  const { isVisible, flipVisibility } = useVisibility(false);
+  const { active: isVisible, toggle: toggleVisibility } = useToggle(false);
```

Comment:
If you return an array, you won't have to alias object keys like this. Just a thought. I would say `setActive` probs doesn't need to be exposed publicly (otherwise why have the toggle), but if you still want to pass it out, making it the last element is probs fine since it would be the least used of the three.